elements/pfe-clipboard/src/pfe-clipboard.scss

```diff
@@ -54,6 +54,7 @@ $LOCAL-VARIABLES: (
   }
 
   &__text--success {
+    font-size: pfe-local(FontSize) !important;
```

Comment:
If you set font-size on the :host, it will push down to all it's children

Reply:
Good call 📞